### ISSUE TITLE
Non-privileged users shouldn't be able to click 'Edit' button in relationship field

### ISSUE THREAD

**Issue by shbchk** [ExpressionEngine/ExpressionEngine#3718]
EE 7.3.11

Video:

[video-convert-1692716828834.webm](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/362816ba-7376-4969-9c01-087232abcf7e)

**Comment by shbchk** [comment]
@intoeetive there are still edit links available in a relationship field:
[video-convert-1696780647263.webm](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/9028e7f3-bca1-4663-ad71-0b237dd26ab3)


EE 7.3.12

**Comment by intoeetive** [comment]
@shbchk am I assuming correctly that the use has _some_ permissions on Filmmamkers channel (e.g. create new entries) but does not have"edit entries" permission? Can you make a screenshot of how the permissions look like?

**Comment by shbchk** [comment]
Yes, your assumption is correct. A member has some permissions: create entries and edit their own entries.

![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/36f906f5-d494-49e9-a9f5-dc8958ec711c)

**Comment by intoeetive** [comment]
This was mostly working correctly as per EE 7.4.11, except for the case where there was AJAX call for loading more entries

Additionally, "edit" link was missing for newly created entries using the modal.

The referenced PR is getting the remaining stuff done.
